Description: Change HOST_NAME_MAX to sysconf(_SC_HOST_NAME_MAX) to support kFreeBSD
Author: James McDonald <james@jamesmcdonald.com>
Bug-Debian: https://bugs.debian.org/782418
--- a/kbfunc.c
+++ b/kbfunc.c
@@ -324,7 +324,7 @@
 	struct menu_q		 menuq;
 	FILE			*fp;
 	char			*buf, *lbuf, *p;
-	char			 hostbuf[HOST_NAME_MAX+1];
+	char			 hostbuf[sysconf(_SC_HOST_NAME_MAX)+1];
 	char			 path[PATH_MAX];
 	int			 l;
 	size_t			 len;
